docs/guides/operator/basic-deployment.adoc

```diff
@@ -100,6 +100,10 @@ When running on OpenShift, with ingress enabled, and with the spec.ingress.class
 The operator will assign a default hostname to the stored version of the CR similar to what would be created by an OpenShift Route without an explicit host - that is ingress-namespace.appsDomain
 If the appsDomain changes, or should you need a different hostname for any reason, then update the Keycloak CR.
 
+NOTE: If you set the `hostname-admin`, or the deprecated `hostname-admin-url`, even if you enable ingress, no ingress will be created specifically for admin access. 
```

Comment:
`hostname-admin-url` has been removed already.
```suggestion
NOTE: If you set the `hostname-admin` even if you enable ingress, no ingress will be created specifically for admin access. 
```

Reply:
> `hostname-admin-url` has been removed already.

Not from the operator #32485

Or do you want to ignore this case for the purposes of this note?

Reply:
Good point. Ok let's leave as is.